vunit/test/report.py

```diff
@@ -18,6 +18,43 @@
 from vunit.ostools import read_file
 
 
+def get_parsed_time(time_in, max_time=0):
+    """
+    Return string representation of input value
+    in hours, minutes and seconds.
+    Inputs:
+      time_in  : Time to convert to string
+      max_time : Longest test time among tests. Optional parameter
+    """
+    time_str = ""
+
+    (minutes, seconds) = divmod(time_in, 60)
+    (hours, minutes) = divmod(minutes, 60)
+    if max(time_in, max_time) >= 3600:
+        # If the longest test took 10 hours or more, pad the string to take this
+        # into account.
+        padding = 5 if (max_time / 3600) % 60 >= 10 else 4
```

Comment:
I suggest this is generalized to any `max_time`. Something like

`len(f"{max_time // 3600} h ")`

Reply:
Nifty! I hadn't seen (nor looked for) that division construct before! I'll use that construction instead.